### PR TITLE
add link to template theme

### PR DESCRIPTION
**Pull request by chrisgrieser**
## Checklist

- [x] I have renamed all attached images with descriptive file names (or I did not include any images)
- [x] Before creating a new note, I searched the vault (or I only edited existing notes)
- [x] (Optional) In case I created a new note in the folder `04 - Guides, Workflows, & Courses`, I added a link to the new note(s) in one of the `for {group X}` overviews ([listed here](https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/%F0%9F%97%82%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses)).

### CODE REVIEW

File: 04 - Guides, Workflows, & Courses/for Theme Designers.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ This note collects resources and guides for beginner and expert theme designers
 - [[CSS 101 - Community Talk]]
 
 ## Tools
-
+- [**Obsidian Theme Template**](https://github.com/obsidian-community/obsidian-theme-template) A template theme to get started
```
Comment (kometenstaub) [inline 04 - Guides, Workflows, & Courses/for Theme Designers.md:37]: Template theme or theme template?

Comment (chrisgrieser) [reply 04 - Guides, Workflows, & Courses/for Theme Designers.md:37]: doesn't both essentially mean the same? 🤔

Comment (claremacrae) [reply 04 - Guides, Workflows, & Courses/for Theme Designers.md:37]: I do think the order matters - but how about making it more explicit - such as "a template project to simplify starting a new theme"

Comment (chrisgrieser) [reply 04 - Guides, Workflows, & Courses/for Theme Designers.md]: okay, I rephrased it.

**Review by claremacrae — APPROVED** [review]
I love the wording - thanks for adding the extra info!

**Pull request merged**